hack/local-up-kubeedge.sh

```diff
@@ -167,27 +184,36 @@ function start_edgecore {
   # if we will use docker as edgecore container runtime
   # we need to change edgecore container runtime from default containerd to docker
   if [[ "${CONTAINER_RUNTIME}" = "docker" ]]; then
-    sed -i 's|containerRuntime: .*|containerRuntime: docker|' ${EDGE_CONFIGFILE}
-    sed -i 's|remoteImageEndpoint: .*|remoteImageEndpoint: unix:///var/run/dockershim.sock|' ${EDGE_CONFIGFILE}
-    sed -i 's|remoteRuntimeEndpoint: .*|remoteRuntimeEndpoint: unix:///var/run/dockershim.sock|' ${EDGE_CONFIGFILE}
+    sed -i 's|imageServiceEndpoint: .*|imageServiceEndpoint: unix:///var/run/cri-dockerd.sock|' ${EDGE_CONFIGFILE}
+    sed -i 's|containerRuntimeEndpoint: .*|containerRuntimeEndpoint: unix:///var/run/cri-dockerd.sock|' ${EDGE_CONFIGFILE}
+  fi
+
+  if [[ "${CONTAINER_RUNTIME}" = "cri-o" ]]; then
+    sed -i 's|imageServiceEndpoint: .*|imageServiceEndpoint: unix:///var/run/crio/crio.sock|' ${EDGE_CONFIGFILE}
+    sed -i 's|containerRuntimeEndpoint: .*|containerRuntimeEndpoint: unix:///var/run/crio/crio.sock|' ${EDGE_CONFIGFILE}
+    sed -i 's|cgroupDriver: .*|cgroupDriver: systemd|' ${EDGE_CONFIGFILE}
+  fi
+
+  if [[ "${CONTAINER_RUNTIME}" = "isulad" ]]; then
+    sed -i 's|imageServiceEndpoint: .*|imageServiceEndpoint: unix:///var/run/isulad.sock|' ${EDGE_CONFIGFILE}
+    sed -i 's|containerRuntimeEndpoint: .*|containerRuntimeEndpoint: unix:///var/run/isulad.sock|' ${EDGE_CONFIGFILE}
```

Comment:
What's the default value of `CONTAINER_RUNTIME` if I don't set it when running local up? @luomengY

Reply:
> What's the default value of `CONTAINER_RUNTIME` if I don't set it when running local up? @luomengY

 The default value for CONTAINER_RUNTIME should be the corresponding containerd. The default container runtime for e2e is also containerd.

Reply:
> > > What's the default value of `CONTAINER_RUNTIME` if I don't set it when running local up? @luomengY
> > 
> > 
> > The default value for CONTAINER_RUNTIME should be the corresponding containerd. The default container runtime for e2e is also containerd.
> 
> It seems the dafault value is `remote`. We can modify it to conatinerd. From v1.16, we don't need the `remote` runtime type any more, as #5299

Yes, the CR corresponding to remote here is containerd, so I will change the remote field to conotainerd here.